x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/planner/QueryFolderFailTests.java

```diff
@@ -22,4 +24,19 @@ public void testPropertyEquationInClauseFilterUnsupported() {
         String msg = e.getMessage();
         assertEquals("Line 1:52: Comparisons against variables are not (currently) supported; offender [parent_process_name] in [==]", msg);
     }
+
+    public void testWildcardNotEnoughArguments() {
```

Comment:
I think there are other error messages to check with `wildcard`: the fact that the field needs to be string and exact and, also, that the "patterns" should be all strings, no?